dashboard-app/src/components/StackedBarChart/util.ts

```diff
@@ -0,0 +1,71 @@
+import { round } from '../../dashboard/dataManipulation/util';
```

Comment:
As a general point, this is a little strange IMO. Dependencies should not, where reasonable, go down the abstraction/generality tree.

To be a little less abstract: components is a directory full of (relatively) generic components, not specific to the requirements of a given feature. Whereas dashboard is concerned with assembling the more generic components into something that satisfies the specific requirements of the features it is trying to implement.

If I delete everything from dashboard, nothing in components (or anywhere else in the app, really, except App.tsx and navigation) should break.

So I think either both the dashboard and the components should rely on some common rounding function in a top level utils, or they should both import from twine-util and specify their own rounding functions (minor preference for the latter because the functionality (rounding) is so trivial).